netplan_cli/cli/state.py

```diff
@@ -193,26 +222,9 @@ def down(self) -> bool:
 
     @property
     def type(self) -> str:
-        match = dict({
-            'bond': 'bond',
-            'bridge': 'bridge',
-            'ether': 'ethernet',
-            'ipgre': 'tunnel',
-            'ip6gre': 'tunnel',
-            'loopback': 'ethernet',
-            'sit': 'tunnel',
-            'tunnel': 'tunnel',
-            'tunnel6': 'tunnel',
-            'wireguard': 'tunnel',
-            'wlan': 'wifi',
-            'wwan': 'modem',
-            'vlan': 'vlan',
-            'vrf': 'vrf',
-            'vxlan': 'tunnel',
-            })
         nd_type = self.nd.get('Type') if self.nd else None
-        if nd_type in match:
-            return match[nd_type]
+        if nd_type := DEVICE_TYPES.get(nd_type):
```

Comment:
todo: the `.get()` will return `None` by default if the value isn't found in `DEVICE_TYPES`, which will still overwrite the `nd_type` variable and break the `logging.warning` below. Returning the original value as a default might work: `DEVICE_TYPES.get(nd_type, nd_type)`